x-pack/docs/en/settings/monitoring-settings.asciidoc

```diff
@@ -5,7 +5,11 @@
 <titleabbrev>Monitoring Settings</titleabbrev>
 ++++
 
-By default, monitoring is enabled. You can adjust how monitoring data is
+By default, monitoring is enabled, but data collection
```

Comment:
can we change it to, "By default, the Monitoring application is enabled, but data collection is disabled." 

I have it as Monitoring with a capital M, for the proper name of the application.